contessa/models.py

```diff
@@ -20,6 +21,24 @@
 from contessa.base_rules import Rule
 from contessa.db import Connector
 
+
+class DataQualityDimension(Enum):
+    QUALITY = "quality"
+    CONSISTENCY = "consistency"
+
+    def __eq__(self, other):
+        if type(other) == "str":
+            return self.value in set(item.value for item in self)
```

Comment:
If I try this, i get "object is not iterable", dunno why 🤔

anyway, i kinda do not understand this. Why, do you compare it with `self.value` and if any of the enums are there you says its equal ? maybe im not getting something, but i thought `self.value`, e.g. is in case DataQualityDimension.QUALITY is "quality". Where is the comparison with `other`? i imagined it that in case of str it would do -> `self.value == other` otherwise super() - or am I wrong ?

also this wont work when turned around `"a" == A.A`, cause its __eq__ from string is executed and if it is implemented for enum (which I guess it is), its not gonna call the other operands __eq__. 

I know it was kinda my idea, I would just rethink the solution a bit more :)

Reply:
:man_facepalming: you are absolutely right, it passed tests but now I see those did not really test it - I put it together with some existing code but it didnt make sense. I simplified it.

As for turning it around - this works because in this case eq method is called on both sides.

I also added tests that check `!=` behaviour.